modules/tinymce/src/plugins/link/test/ts/browser/LinkContextMenuTest.ts

```diff
@@ -0,0 +1,31 @@
+import { FocusTools, Keys } from '@ephox/agar';
+import { describe, it } from '@ephox/bedrock-client';
+import { SugarDocument } from '@ephox/sugar';
+import { TinyHooks, TinyUiActions } from '@ephox/wrap-mcagar';
+
+import Editor from 'tinymce/core/api/Editor';
+import Plugin from 'tinymce/plugins/link/Plugin';
+
+describe('browser.tinymce.plugins.link.LinkContextMenuTest', () => {
+  const hook = TinyHooks.bddSetup<Editor>({
+    plugins: 'link',
+    toolbar: 'link',
+    base_url: '/project/tinymce/js/tinymce',
+  }, [ Plugin ]);
+
+  const pAssertFocusOnItem = (label: string, selector: string) =>
+    FocusTools.pTryOnSelector(`Focus should be on: ${label}`, SugarDocument.getDocument(), selector);
+
+  const pressDownArrowKey = (editor: Editor) => TinyUiActions.keydown(editor, Keys.down());
+
+  it('TBA: Context menu show up on link', async () => {
```

Comment:
```suggestion
  it('TINY-2293: Context menu show up on link', async () => {
```
We normally try to include the Jira number so we have something to track back to why this was added.